### ISSUE TITLE
PUAE does not support chd

### ISSUE THREAD

**Issue by sionisaac** [TeamPandory/pandory500#9]
I love this mod for A500 mini but I have found that none of my chd CDTV or CD32 games work the core reports "unsupported file format CHD", if I go to core information it says the format is supported by the core.  I know that my CHD's are good because they work with zero issues on retroarch for windows/linux/android/retropie/mac.  Support for chd files so I can use my complete collection of CDTV and CD32 games on it and ADF/IPF support.  Is there likely to be an update that fixes this issue?

**Comment by sionisaac** [comment]
Edit to original post.  PSX/SEGACD/PCFX/PCE CD cores all support CHD files and I've tested them and they work fine.  Final Burn NeoGeoCD subsystem does not support CHD only BIN/CUE.  This seems to be a problem with the amiga core PUAE.  The Windows/Linux/Android/Mac versions of exactly the same core do not have issues with CHD.  Is it possible to update the core manually using a nightly core?

**Comment by TeamPandory** [comment]
Hey there!
Thankyou for your kind words..  Personally I haven't tried CHD but you've now got me interested :-D
It may be because the .nfo files need changing slightly-  I found this on another github page.
<img width="688" alt="image" src="https://user-images.githubusercontent.com/45140115/178170090-8a6e555c-0ad5-4068-8933-ea18ab763e5d.png">

**Comment by sionisaac** [comment]
The core information page says chd is supported in list of supported file types.

**Comment by sionisaac** [comment]
I used the linux terminal to copy all the core info files to my memory stick and opened them in notepad++ the info files have support for chd.  I did a compare with the info files from windows retroarch and pandory.  I tried to use a nightly core by changing the core directory to a sub folder off the root of my memory stick and put puae_libretro.so from the ARMF branch of nightlies in there and guess what.  It does exactly the same thing "unsupported file format CHD".  This is really annoying me now I know that I can convert the chd's back to bin/cue but that is not the point all other retroarch systems support chd for the PUAE core.  I even tried it on KMFDMANIC build for my PSC and it works fine on that too.  I take it this will not get fixed as no one else seems to have a problem with it.
